.github/workflows/ESLint.yml

```diff
@@ -0,0 +1,11 @@
+name: ESLint Check
+on: push
```

Comment:
Can you please change it to push and pull request both, so that we can add this testcase?

Reply:
Also, I think you have changed the folder for main app, but it looks good, please add the test on PR, so that we can test, if it is working or not. 
Thanks.

Reply:
> Can you please change it to push and pull request both, so that we can add this testcase?

Added pull request trigger in Action.


> Also, I think you have changed the folder for main app, but it looks good, please add the test on PR, so that we can test, if it is working or not. Thanks.

I did this because yarn required the package.json file to be available to the Github Action Runner.

Reply:
No problem, I think we can change it like that.